web/src/lib/elements/ui/dataTable/data-table.svelte

```diff
@@ -44,7 +44,7 @@
   export let additionalId: string = '';
   export let sortable = true;
   export let addButton = true;
-  export let pageSize = 15;
+  export let pageSize = 2;
```

Comment:
I don't think a default size of 2 is feasible

Reply:
You're right. I used that for testing but forgot removing it.